pkg/ui/distoss/distoss.go

```diff
@@ -11,3 +11,17 @@
 // Package distoss embeds the assets for the OSS version of the web UI into the
 // Cockroach binary.
 package distoss
+
+import (
+	"embed"
+
+	"github.com/cockroachdb/cockroach/pkg/ui"
+)
+
+//go:embed assets
```

Comment:
Just making sure, this technique can work with Bazel? I've never seen it before.

Reply:
Go Source rule includes `embedsrcs` field which allows to deal with `go:embed` directive:
https://github.com/bazelbuild/rules_go/blob/master/go/providers.rst#gosource